### PR TITLE
chore: add <start> script for platform properties provider

### PR DESCRIPTION
**Pull request by MaxTru**
* This allows to easily play around with the platform properties
  provider

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "dev": "npm test -- --auto-watch --no-single-run",
     "test": "karma start karma.config.js",
     "test:build": "mocha --reporter=spec --recursive test/distro",
-    "start": "cross-env SINGLE_START=1 npm run dev",
+    "start": "npm run start:cloud",
+    "start:cloud": "cross-env SINGLE_START=1 PROP_PROVIDER=CLOUD npm run dev",
```
Comment (pinussilvestrus) [inline package.json:16]: Two flags seem to be one too much.

Maybe we don't need the `SINGLE_START` flag any more. Or we simply do it [like in `camunda-bpmn-js`](https://github.com/camunda/camunda-bpmn-js/blob/main/package.json#L14).

```sh
SINGLE_START=cloud
SINGLE_START=platform
```

Comment (MaxTru) [reply package.json]: Yeah, very good point. I adjusted it accordingly

File: test/spec/BpmnPropertiesPanelRenderer.spec.js [hunk 2]
```diff
@@ -21,14 +21,18 @@ import Modeler from 'bpmn-js/lib/Modeler';
...
+import CamundaModdle from 'camunda-bpmn-moddle/resources/camunda';
+import CamundaModdleExtension from 'camunda-bpmn-moddle/lib';
+
 import ZeebeModdle from 'zeebe-bpmn-moddle/resources/zeebe';
 import ZeebeModdleExtension from 'zeebe-bpmn-moddle/lib';
 
 import ExamplePropertiesProvider from './extension/ExamplePropertiesProvider';
 
-const singleStart = window.__env__ && window.__env__.SINGLE_START;
+const singleStart = (window.__env__ && window.__env__.SINGLE_START) || 'cloud';
```
Comment (pinussilvestrus) [inline test/spec/BpmnPropertiesPanelRenderer.spec.js:35]: That leads to `singleStart` is never undefined and we always only execute one test:

https://github.com/bpmn-io/bpmn-properties-panel/pull/15/checks?check_run_id=3047276493#step:6:31
image

Comment (pinussilvestrus) [reply test/spec/BpmnPropertiesPanelRenderer.spec.js:35]: Do we need this default? If nothing is configured, no single start.

Comment (MaxTru) [reply test/spec/BpmnPropertiesPanelRenderer.spec.js]: ups sorry, fixed that.

Thanks for the check

**Review by pinussilvestrus — APPROVED** [review]
👍

**Pull request merged**